test/config_test/examples/mnist.test.yml

```diff
@@ -2,7 +2,7 @@ authorName: nni
 experimentName: default_test
 maxExecDuration: 5m
 maxTrialNum: 2
-trialConcurrency: 2
+trialConcurrency: 1
```

Comment:
why change this?

Reply:
This is a temp workaround for this issue:  https://github.com/Microsoft/nni/issues/621